package.json

```diff
@@ -1,13 +1,14 @@
 {
-  "name": "nodejs-katas",
+  "name": "helloworld",
   "version": "1.0.0",
   "description": "Basic Node.js exercises",
   "main": "index.js",
   "scripts": {
     "start": "node index.js",
-    "lint": "eslint test",
-    "test": "mocha test/**/*-test.js",
-    "watch": "npm test -- -w"
+    "lint": "eslint ./test ./src",
+    "test": "npm run lint && mocha ./test/test.js -w",
```

Comment:
We shouldn't use the `-w` switch in the `test` script. Otherwise, tests on CI will hang, like just now.